src/app/app.module.ts

```diff
@@ -28,14 +26,13 @@ import { AngularFireAuth } from "@angular/fire/auth";
     IonicModule.forRoot(),
     AppRoutingModule,
     AngularFireModule.initializeApp(environment.firebase),
+    AngularFireAuthModule,
     AngularFirestoreModule,
   ],
   providers: [
     StatusBar,
     SplashScreen,
     Camera,
-    AuthService,
-    AngularFireAuth,
```

Comment:
No need for explicit providers in most cases; injecting AuthService directly into the constructor for a page that needs it works just as well.